WORKSPACE

```diff
@@ -220,6 +221,14 @@ pypi_repository(
     version = "3.4.0",
 )
 
+pypi_wheel_repository(
```

Comment:
Sure, I will make the test, however it might be useful to keep the subrepository in case we want to work on the c++ module later (only for development purposes) we could also remove the submodule initialization from the configure script (to be considered). 

Otherwise we can move the whole c++ module to an other branch and clean the related part from the develop branch (submodule, c++ target)